src/ui/settings/ViewGuide.qml

```diff
@@ -253,4 +253,11 @@ Item {
     }
 
     Component.onCompleted: VPNCloseEventHandler.addView(root)
+
+    Component.onDestruction: {
+        VPN.recordGleanEventWithExtraKeys("guideClosed",{
+                                          "id": guide.id,
+                                          "duration": `${((new Date().getTime() - timeOfOpen) / 1000)} seconds`
```

Comment:
This should be a number and the key should indicate the units e.g. "duration_s": 1234

As a string analysis will be a big pain.

Reply:
I don't think you need to worry about handling that. Just do the math. You could also just make it milliseconds so "duration_ms" and then you don't need to do the division.